### DISCUSSION TITLE
Links won’t open in edge

### DISCUSSION THREAD

**Discussion by willuhmjs** [community/community#11379] [Mobile]
I selected a link. It opened edge but did not open the link, just the previous tab I was in.

**Comment by mxie** [comment]
Thanks for reaching out to us @willuhm-js. Where the link was supposed to take you to? For example, an issue, a pull request, a comment, or something else?

**Reply by willuhmjs** [reply]
It's the link in my profile / bio section. Tapping it in the mobile client launches Edge and reloads the tab I left off on. It doesn't actually open the link.
